src/index.html

```diff
@@ -5,4 +5,151 @@
 order: 1
 ---
 
-<p>Add the HTML for your homepage here.</p>
+<iframe src="https://www.youtube.com/embed/z_jefhcVRAk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="about">
+    <div class="img">
+        <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fabout.png?v=1605143407347" alt="Dancing Girl">
+    </div>
+    <div class="text">
+        <h1>About MSDC</h1>
+        <p>Originally established in 1992, Morton Street Dance Center is one of Baltimore's best and most comprehensive dance schools.</p>
+        <button class="more">Learn More</button>
+    </div>
+</div>
+<div class="offerings">
+        <h1>Our CLass Offerings</h1>
+    <ul>
+        <li class="ballet">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fballet.png?v=1605143408910" alt="Ballet">
+            <h2>Ballet</h2>
+        </li>
+        <li class="pointe">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fpointe.png?v=1605143399024" alt="Pointe">
+            <h2>Pointe</h2>
+        </li>
+        <li class="modern">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fmodern.png?v=1605143401181" alt="Modern">
+            <h2>Modern</h2>
+        </li>
+        <li class="tap">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Ftap.png?v=1605143407247" alt="Tap">
+            <h2>Tap</h2>
+        </li>
+        <li class="jazz">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fjazz.png?v=1605143407596" alt="Jazz">
+            <h2>Jazz</h2>
+        </li>
+        <li class="hiphop">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fhiphop.png?v=1605143407749" alt="Hip-Hop">
+            <h2>Hip-Hop</h2>
+        </li>
+        <li class="african">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fafrican.png?v=1605143399790" alt="African">
+            <h2>African</h2>
+        </li>
+        <li class="little-ones">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Flittleones.png?v=1605143402975" alt="little-ones">
+            <h2>Little Ones</h2>
+        </li>
+        <li class="repertory">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Frepertory.png?v=1605143401041" alt="repertory">
+            <h2>Repertory</h2>
+        </li>
+        <li class="adult-classes">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fadultclasses.png?v=1605143403559" alt="dult Classes">
+            <h2>Adult Classes</h2>
+        </li>
+    </ul>
+</div>
+<div class="Register">
+    <h1>2020 Fall Registration</h1>
+    <button class="accordion">Step 1</button>
```

Comment:
All accordion `button` elements must have a set `type` attribute, like so:
```suggestion
    <button class="accordion" type="button">Step 1</button>
```